src/generate-elements.ts

```diff
@@ -169,7 +180,8 @@ class ElementGatherer implements AnalysisVisitor {
       }
     } else {
       this._elementPaths.set(element, pathToElement);
-      const elem = serializeElementDescriptor(element, pathToElement);
+      const elem =
```

Comment:
Good point. Expanded `element` to `elementDescriptor` and `elem` to `element` here to clarify that ambiguity.